modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -830,6 +833,42 @@ public void testAddObjectEntryWithObjectValidationRule() throws Exception {
 			).build());
 
 		Assert.assertNotNull(objectEntry);
+
+		_objectValidationRuleLocalService.addObjectValidationRule(
+			TestPropsValues.getUserId(),
+			_objectDefinition.getObjectDefinitionId(), true,
+			ObjectValidationRuleConstants.ENGINE_TYPE_DDM,
+			LocalizedMapUtil.getLocalizedMap("Error"),
+			LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString()),
+			"numberOfBooksWritten > externalReferenceCode");
+
+		try {
+			_addObjectEntry(
+				HashMapBuilder.<String, Serializable>put(
+					"birthday", "2000-12-25"
+				).put(
+					"emailAddressRequired", "bob@liferay.com"
+				).put(
+					"listTypeEntryKeyRequired", "listTypeEntryKey1"
+				).put(
+					"numberOfBooksWritten", "123"
+				).build());
+
+			Assert.fail();
+		}
+		catch (ModelListenerException modelListenerException) {
+			Assert.assertTrue(
+				modelListenerException.getCause() instanceof
+					ObjectValidationRuleEngineException.InvalidScript);
+
+			ObjectValidationRuleEngineException.InvalidScript invalidScript =
+				(ObjectValidationRuleEngineException.InvalidScript)
+					modelListenerException.getCause();
+
+			Assert.assertEquals(
+				"there-was-an-error-validating-your-data",
+				invalidScript.getMessageKey());
+		}
```

Comment:
@p-albuquerque can we simplify to:

```
catch (ModelListenerException modelListenerException) {
	String message = modelListenerException.getMessage();

	Assert.assertTrue(message.contains("Error"));
}
```
and follow the same pattern in the other tests?